src/styles/utils/_functions.scss

```diff
@@ -0,0 +1,7 @@
+@function calendarWidth(
+  $square-size, $elements-in-rows, $gap, $container-padding) {
```

Comment:
Consider renaming the function to something more specific, like 'calculateCalendarWidth', to make it clear what the function does.